circle.yml

```diff
@@ -1,3 +1,14 @@
-machine:
-  node:
-    version: 8.4.0
\ No newline at end of file
+version: 2
+jobs:
+  build:
+    working_directory: ~/tmp
+    docker:
+      - image: circleci/node:8
+      - image: circleci/postgres:9.6.2-alpine
+        environment:
+          - POSTGRES_USER=ubuntu
+          - POSTGRES_DB=gauntlet-development

```

Comment:
Aren't we gonna use the `ormconfig.test.json` credentials?

Reply:
I am not sure on how to implement that. But it is probably good if we use the ormconfig.test.json